### PR TITLE
Add VueToronto banner to the docs

feat(*): add vuetoronto banner (#1456)

### LINKED ISSUE #1456: Add VueToronto banner to the docs
New updates on head repo.
https://github.com/vuejs/docs/commit/58c4b8992527c252b18b834e4712ebe682e43db2

### PR DESCRIPTION
**Pull request by edimitchel**
resolves #1456
Cherry picked from https://github.com/vuejs/docs/commit/58c4b8992527c252b18b834e4712ebe682e43db2

### CODE REVIEW

File: .vitepress/theme/components/Banner.vue [hunk 1]
```diff
@@ -22,16 +22,31 @@ function dismiss() {
 
 <template>
   <div class="banner" v-if="open">
-    <a target="_blank"></a>
+    <p class="vt-banner-text">
+      <span class="vt-text-primary">VueConf Toronto</span>
+      <span class="vt-tagline"> - Join the premier Vue.js conference</span>
```
Comment (tisma95) [inline .vitepress/theme/components/Banner.vue:27]:
```suggestion
      <span class="vt-tagline"> - Rejoignez la première conférence Vue.js</span>
```

File: .vitepress/theme/components/Banner.vue [hunk 2]
```diff
@@ -22,16 +22,31 @@ function dismiss() {
...
+        href="https://vuetoronto.com/?utm_source=vuejs&utm_content=top_banner"
+      >
+        Register
+      </a>
+    </p>
     <button @click="dismiss">
       <VTIconPlus class="close" />
     </button>
+    <p class="vt-banner-text vt-coupon">
+      <span class="vt-text-primary">Use code</span> VUEJS
```
Comment (tisma95) [inline .vitepress/theme/components/Banner.vue:41]:
```suggestion
      <span class="vt-text-primary">Utiliser le code</span> VUEJS
```

File: .vitepress/theme/components/Banner.vue [hunk 3]
```diff
@@ -22,16 +22,31 @@ function dismiss() {
...
+      <span class="vt-text-primary">to get 15% off</span>
```
Comment (tisma95) [inline .vitepress/theme/components/Banner.vue:42]:
```suggestion
      <span class="vt-text-primary">pour obtenir 15 % de réduction</span>
```

**Review by tisma95 — CHANGES_REQUESTED** [review]
Translation to do

File: .vitepress/theme/components/Banner.vue [hunk 4]
```diff
@@ -22,16 +22,31 @@ function dismiss() {
...
+    <p class="vt-banner-text">
+      <span class="vt-text-primary">VueConf Toronto</span>
+      <span class="vt-tagline"> - Rejoignez la première conférence Vue.js</span>
+      | 18-20 Nov 2024 <span class="vt-place"> - Toronto, Canada</span>
+      <a
+        target="_blank"
+        class="vt-primary-action"
+        href="https://vuetoronto.com/?utm_source=vuejs&utm_content=top_banner"
+      >
+        Register
```
Comment (edimitchel) [inline .vitepress/theme/components/Banner.vue:34]:
```suggestion
        S'inscrire
```

**Pull request merged**